phpBB/styles/prosilver/template/overall_header.html

```diff
@@ -60,7 +60,7 @@
 <!-- IF S_COOKIE_NOTICE -->
 	<link href="{T_ASSETS_PATH}/cookieconsent/cookieconsent.min.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet">
 <!-- ENDIF -->
-
+	<link href="{T_THEME_PATH}/jquery.atwho.css" rel="stylesheet">
```

Comment:
You only need this on the pages where at.js is used, so you should add a IF statement to only include it on those pages.